general-info.qmd

```diff
@@ -21,3 +21,52 @@ We work in the Bioscience Research Laboratory (BSRL)
     -   TUGO is great for getting around campus
 -   Parking Easiest parking is the garage two blocks east of BSRL: \$2 hourly or \$8 daily.
 -   Public Transportation
+
+## Group Communication
```

Comment:
split out into a new chapter `group-communication.qmd` in Group Procedures section